pkg/amphoracontrollers/flavors.go

```diff
@@ -230,7 +238,7 @@ func ensureFlavors(osclient *openstack.OpenStack, log *logr.Logger, instance *oc
 			log.Info(fmt.Sprintf("Creating Octavia flavor profile \"%s\"", flavorProfileCreateOpts.Name))
 			fp, err := flavorprofiles.Create(lbClient, flavorProfileCreateOpts).Extract()
 			if err != nil {
-				return "", err
+				return "", fmt.Errorf("error creating flavor profiles: %w", err)
```

Comment:
I think we need to have a fallback when a flavorprofile cannot be created (for instance if the image tag doesn't exist).
we could ignore the error and continue, or maybe report an error only if none of the flavors were created successfully.

Right now, this patch will block the promotion pipeline with upstream content

Reply:
Thanks for the suggestion. I've pushed an update that should implement that idea.